src/check/parse/IR.zig

```diff
@@ -2146,19 +2146,22 @@ pub const NodeStore = struct {
                     // const token = ir.tokens.tokens.get(str.token);
                     // std.debug.print("TOKEN: {}\n", .{token});
 
+                    var sexpr_str = sexpr.Expr.init(gpa, "string");
                     for (str.parts) |part_id| {
                         const part_expr = ir.store.getExpr(part_id);
-                        return part_expr.toSExpr(gpa, env, ir);
+                        var part_sexpr = part_expr.toSExpr(gpa, env, ir);
+                        sexpr_str.appendNodeChild(gpa, &part_sexpr);
                         // std.debug.print("PART: {}\n", .{part_expr});
                     }
 
-                    @panic("not implemented");
+                    return sexpr_str;
                 },
-                .string_part => |_| {
+                .string_part => |sp| {
```

Comment:
This branch could be unreachable if we wanted to but I guess better to tolerate minor problems so they can be visible?